scripts/scil_volume_reshape.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to reshape a volume to match the resolution of another
+reference volume or to the resolution specified as in argument. The resulting
+volume will be centered in world space wrt. the reference volume or the
+specified resolution.
+
+This script will pad or crop the volume to match the desired shape.
+To interpolate, use scil_volume_resample.py.
```

Comment:
Could you write that
scil_volume_resample.py = interpolate to a specific voxel size
scil_reslice_to_reference = copy the croping and voxel size of another image in world space
scil_reshape = pad or crop from center to have specific dimension
scil_crop = Remove extra black space around any shape (bounding box)

And maybe we need these 4 distinctions in all 4 scripts?
(This should also go in scil_documentation at some point)